tests/visualization_tests/test_timeline.py

```diff
@@ -154,7 +154,11 @@ def test_get_timeline_plot_with_killed_running_trials(
     plot_timeline: Callable[..., Any], waiting_time: float
 ) -> None:
     def _objective_with_sleep(trial: optuna.Trial) -> float:
-        time.sleep(0.1)
+        sleep_start_datetime = datetime.datetime.now()
+        # Spin waiting is used here because high accuracy is necessary even in weak VM.
```

Comment:
```suggestion
        # Spin waiting is used here because high accuracy is necessary even in weak VM.
        # Please check the motivation of the bugfix in https://github.com/optuna/optuna/pull/5549/
```